packages/material-ui-lab/src/TreeItem/TreeItem.js

```diff
@@ -16,38 +16,63 @@ export const styles = theme => ({
     padding: 0,
     outline: 0,
     WebkitTapHighlightColor: 'transparent',
-    '&:focus > $content': {
-      backgroundColor: theme.palette.grey[400],
+    '&:focus > $content $label': {
+      backgroundColor: theme.palette.action.hover,
+    },
+    '&$selected > $content $label': {
+      backgroundColor: fade(theme.palette.primary.main, theme.palette.action.selectedOpacity),
+    },
+    '&$selected > $content $label:hover, &$selected:focus > $content $label': {
```

Comment:
I wonder about the following options to simplify the overrides:

1. We could, in theory, use:
```suggestion
    '&$selected > div $label:hover, &$selected:focus > div $label': {
```
2. However, and maybe it's even better. Do we need to apply the pseudo-classes to the root element? What if we apply it to the content element directly? I can't think of any significant downsides.

I like 2.